client/app/pages/dashboards/DashboardListEmptyState.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BigMessage } from '@/components/BigMessage';
+import { NoTaggedObjectsFound } from '@/components/NoTaggedObjectsFound';
+import { EmptyState } from '@/components/empty-state/EmptyState';
+
+export default function DashboardListEmptyState({ page, searchTerm, selectedTags }) {
+  if (searchTerm !== '') {
+    return (
+      <BigMessage message="Sorry, we couldn't find anything." icon="fa-search" />
+    );
+  }
+  if (selectedTags.length > 0) {
+    return (
+      <NoTaggedObjectsFound objectType="dashboards" tags={selectedTags} />
+    );
+  }
+  switch (page) {
+    case 'favorites': return (
+      <BigMessage message="Mark dashboards as Favorite to list them here." icon="fa-star" />
+    );
+    default: return (
+      <EmptyState
+        icon="zmdi zmdi-view-quilt"
+        description="See the big picture"
+        illustration="dashboard"
+        helpLink="https://help.redash.io/category/22-dashboards"
```

Comment:
For future consideration - links to external sites are better kept in a `constants` config file, so that updating them can be made easily and safely.